src/mapml.css

```diff
@@ -426,4 +426,13 @@ summary {
   white-space: nowrap;
   text-align: center;
   padding: 2px;
+}
+
+.mapml-outline {
+  outline-style: auto;
+  outline-offset: -1px;
+  outline-color: #44A7CB;
```

Comment:
I would prefer if this didn't set a particular `outline-color`, to keep the component as unopinionated as possible, but maybe that was that problematic for some reason?

Reply:
No just thought the default color (black) looked bad, just a color preference, it's also a similar color Google Map's uses to outline but it can be removed.